src/sass/components/_footer.scss

```diff
@@ -1,67 +1,105 @@
-.m-footer {
-  flex-shrink: 0;
-  background-color: var(--footer-background-color);
+.footer {
+  background-color: #003262;
+  padding: 15px;
 }
 
-.m-footer__content {
-  color: $white;
-  text-align: center;
-  padding: 30px $mobile-space;
+.footer-main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
 
-  @include respond-to('medium') {
-    padding: 70px 0;
-    max-width: 330px;
-    margin: 0 auto;
-  }
+.footer-left {
+  margin: 5px;
+}
+
+.footer-title, .footer-subtitle {
+  font-weight: bolder;
+  color: #fff;
+}
+
+.footer-title {
+  font-size: 24px;
 }
 
-.m-footer-copyright {
-  font-size: 0.875rem;
-  letter-spacing: 0.3px;
-  line-height: 1.75rem;
-  margin-bottom: $mobile-space;
+.footer-subtitle {
+  font-size: 16px;
+}
+
+.footer-social {
+  margin: 15px 0px;
+}
 
-  span {
-    display: block;
+.footer-social a {
+  font-size: 25px;
+  color: #fff;
+  margin-right: 15px;
+}
 
-    @include respond-to('medium') {
-      display: inline;
-    }
+.footer-social a:last-of-type {
+  margin-right: 0px;
+}
 
-    &:nth-child(2) {
-      display: none;
+.footer-mailing-list button {
```

Comment:
There seems to be a small border-radius on the image, perhaps adding `border-radius: 2px;` would work.